.github/workflows/multicluster.yaml

```diff
@@ -83,6 +83,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9
         with:
+          # renovate: datasource=golang-version depName=go
           go-version: 1.20.2
```

Comment:
complete drive-by but I learned recently that we should probably be quoting these versions: https://xeiaso.net/blog/go-1.20-yaml

Reply:
Good point. I guess we're not affected by this though because we always specify complete 1.x.y versions, not 1.x versions which might be (mis)interpreted as float. I'll leave the version as is for the sake of this PR and will send a follow up fix.